x-pack/plugin/rollup/src/main/java/org/elasticsearch/xpack/downsample/RollupShardIndexer.java

```diff
@@ -131,6 +135,10 @@ class RollupShardIndexer {
 
     public DownsampleIndexerAction.ShardDownsampleResponse execute() throws IOException {
         long startTime = System.currentTimeMillis();
+        task.setTotalShardDocCount(searcher.getDirectoryReader().numDocs());
+        task.setRollupShardIndexerStatus(RollupShardIndexerStatus.STARTED);
+        task.setIndexStartTimestampMillis(searchExecutionContext.getIndexSettings().getTimestampBounds().startTime());
+        task.setIndexEndTimestampMillis(searchExecutionContext.getIndexSettings().getTimestampBounds().endTime());
```

Comment:
Since we have other timings like `lastIndexingTimestamp` it is useful, in my opinion to have start and end time in the same place to have an idea of where the task is.

Regarding the end time being updated...I thought it would not change anymore after the index is made read-only and rolls over....am I wrong?